web/templates/move_topic_to_stream.hbs

```diff
@@ -1,26 +1,50 @@
 {{#unless (or from_message_actions_popover only_topic_edit)}}
-<p class="white-space-preserve-wrap">{{#tr}}Move all messages in <strong>{topic_name}</strong>{{/tr}} to:</p>
+<p>
+    {{#tr}}
+    You are moving all messages from <z-stream></z-stream>
+    {{#*inline "z-stream"}}<b>{{> inline_decorated_stream_name stream=stream show_colored_icon=true}}</b>{{/inline}}
+    {{/tr}}
+    <i class="fa fa-angle-right" aria-hidden="true"></i>
+    {{#tr}}<b>{topic_name}.</b>{{/tr}}
+</p>
 {{/unless}}
 <form id="move_topic_form">
     {{#if only_topic_edit }}
     <p>{{t "Rename topic to:" }}</p>
     {{else if from_message_actions_popover}}
-    <p>{{t "Move messages to:" }}</p>
+    <p>
+        {{#tr}}
+        You are moving messages from <z-stream></z-stream>
+        {{#*inline "z-stream"}}<b>{{> inline_decorated_stream_name stream=stream show_colored_icon=true}}</b>{{/inline}}
+        {{/tr}}
+        <i class="fa fa-angle-right" aria-hidden="true"></i>
+        {{#tr}}<b>{topic_name}.</b>{{/tr}}
+    </p>
```

Comment:
I think this block belongs with the one above it.

Reply:
Maybe we could structure this template in general. The `only_topic_edit` conditional looks like it is being used multiple times – being used when only topic renaming is possible. We could consolidate the elements to show for `only_topic_edit`.